packages/dds/ordered-collection/src/consensusOrderedCollection.ts

```diff
@@ -237,7 +238,11 @@ export class ConsensusOrderedCollection<T = any>
 				opName: "release",
 				acquireId,
 			}).catch((error) => {
-				this.runtime.logger.sendErrorEvent({ eventName: "ConsensusQueue_release" }, error);
+				const logger = createChildLogger({
+					logger: this.runtime.logger,
+					namespace: "fluid:ConsensusOrderedCollectionDataStoreRuntime",
+				});
```

Comment:
Same as a comment above; we shouldn't create the child logger every single time we need to log an event. Can you look for a better place to do it in this case?

Reply:
`createChildLogger` converting `Base` type logger to `Ext` type only requiring the original logger and a namespace. Although I'm using it everywhere, it's kinda lightweight converting function already.

The `ConsensusOrderedCollection` is a @alpha dds and its runtime is @public. It use this logger to log error [supported by `Ext` type](https://github.com/microsoft/FluidFramework/pull/15667). I think it should be right place to put on a convert function here.

Reply:
It's true that `createChildLogger` is kind of lightweight but if it's straightforward to not call it multiple times, that's still better. Similar to the other place where we moved it, in `ConsensusOrderedCollection`'s  constructor we have access to `this.runtime.logger` so we could convert it to an Ext logger there once, and here directly reference the converted logger.

Also note that we don't want to add new namespaces where there were none before. `createChildLogger` is fine receiving an empty namespace, so we shouldn't add one here in order to maintain whatever namespace these logs are currently getting.

Reply:
Updated